source/documentation/overview/benefits.md

```diff
@@ -4,15 +4,15 @@ Right now, development teams spend considerable time and money setting up all th
 
 Your development team can use PaaS to deploy and run government applications written in a range of languages and web frameworks. You’ll simply need to push your source code to the PaaS environment and the code will be compiled and deployed for you.
 
-PaaS makes user management easy by providing development teams with privilege separation options, eg an ‘admin’ user will be able to assign other team members certain permissions from their PaaS console.
+PaaS makes user management easy by providing development teams with privilege separation options, for example an ‘admin’ user will be able to assign other team members certain permissions from their PaaS console.
 
 PaaS is deployed to multiple availability zones, making it resilient and accredited for information up to OFFICIAL.
 
 The platform itself will be supported 24/7 by GDS, although this does not include application support.
 
 Features of the PaaS platform currently include:
 
-*   backing service support for PostgreSQL, MySQL and MongoDB
+*   support for multiple backing services
```

Comment:
Should we still include some examples (like our most used services) as examples? This is an overview where this might be important.